pandokia/default_config.py

```diff
@@ -289,4 +289,4 @@ def complex_readpass():
 #
 # END OF CONFIGURATION
 #
-flagok_file = "/eng/ssb/tests/pdk_updates/%s.ok"
+flagok_file = "/eng/etc/test/pdk_updates/%s.ok"
```

Comment:
fixing this line, though this line was NOT the problem - this file is just an example file.  see the etc-controller PR